frontend/main.py

```diff
@@ -248,15 +251,15 @@ def reject_correction(queue_name, correction_id, email_template, version, userna
 
 
 def approve_correction(
-    queue_name, correction_id, email_template, version, username, corrected_val
+    queue_name, correction_id, email_template, version, username, corrected_val, attribute
```

Comment:
approve method should only accept values that either help find the record in db or the values that are being changed. Backend already knows what attribute this correction was for, it shouldn't need this to be sent again from frontend again.

Reply:
could you explain how backend knows what attribute to be corrected if we only have corrected_val that contains either grade or hadithText from frontend? jazakAllah khair

Reply:
https://github.com/sunnah-com/corrections/blob/master/frontend/scripts/init-dynamodb.sh#L33

The correction record in dynamodb already has the attribute being corrected. We only need correction_id to load the correction from the database and we can figure out what attribute was corrected.

Reply:
yes I just realized it too, thank you. fix is in progress 🙏🏽

Reply:
updated